Packs/FireEyeCM/Integrations/FireEyeCM/FireEyeCM.py

```diff
@@ -0,0 +1,596 @@
+                                       appliance_id: str, limit: int) -> Dict[str, str]:
+        params = {
+            'start_time': start_time,
+            'end_time': end_time,
+            'limit': limit
+        }
+        if from_:
+            params['from'] = from_
+        if subject:
+            params['subject'] = subject
+        if appliance_id:
+            params['appliance_id'] = appliance_id
+
+        return self._http_request(method='GET', url_suffix='emailmgmt/quarantine', params=params, resp_type='json')
+
+    @logger
+    def release_quarantined_emails_request(self, sensor_name: str, queue_ids: list):
+        return self._http_request(method='POST',
+                                  url_suffix='emailmgmt/quarantine/release',
+                                  params={'sensorName': sensor_name},
+                                  json_data={"queue_ids": queue_ids},
+                                  resp_type='resp')
+
+    @logger
+    def delete_quarantined_emails_request(self, sensor_name: str, queue_ids: list):
+        return self._http_request(method='POST',
+                                  url_suffix='emailmgmt/quarantine/delete',
+                                  params={'sensorName': sensor_name},
+                                  json_data={"queue_ids": queue_ids},
+                                  resp_type='resp')
+
+    @logger
+    def download_quarantined_emails_request(self, sensor_name: str, queue_id: str, timeout: str):
+        self._headers.pop('Accept')  # returns a file, hence this header is disruptive
+        return self._http_request(method='GET',
+                                  url_suffix=f'emailmgmt/quarantine/{queue_id}',
+                                  params={'sensorName': sensor_name},
+                                  resp_type='content',
+                                  timeout=timeout)
+
+    @logger
+    def get_reports_request(self, report_type: str, start_time: str, end_time: str, limit: str, interface: str,
+                            alert_id: str, infection_type: str, infection_id: str, timeout: int):
+        params = {
+            'report_type': report_type,
+            'start_time': start_time,
+            'end_time': end_time
+        }
+        if limit:
+            params['limit'] = limit
+        if interface:
+            params['interface'] = interface
+        if alert_id:
+            params['id'] = alert_id
+        if infection_type:
+            params['infection_type'] = infection_type
+        if infection_id:
+            params['infection_id'] = infection_id
+
+        return self._http_request(method='GET',
+                                  url_suffix='reports/report',
+                                  params=params,
+                                  resp_type='content',
+                                  timeout=timeout)
+
+
+@logger
+def to_fe_datetime_converter(time_given: str = 'now') -> str:
```

Comment:
BTW, are we using this on other integrations also?

Reply:
not yet. will be used in https://github.com/demisto/content/pull/13041, will consolidate to an APIModule.